src/core/settings/qgssettingsregistrycore.h

```diff
@@ -57,17 +56,8 @@ class CORE_EXPORT QgsSettingsRegistryCore : public QgsSettingsRegistry
     //! Settings entry digitizing line width
     static const inline QgsSettingsEntryInteger settingsDigitizingLineWidth = QgsSettingsEntryInteger( QStringLiteral( "line_width" ), QgsSettings::Prefix::QGIS_DIGITIZING, 1 );
 
-    //! Settings entry digitizing line color red
-    static const inline QgsSettingsEntryInteger settingsDigitizingLineColorRed = QgsSettingsEntryInteger( QStringLiteral( "line_color_red" ), QgsSettings::Prefix::QGIS_DIGITIZING, 255 );
-
-    //! Settings entry digitizing line color green
-    static const inline QgsSettingsEntryInteger settingsDigitizingLineColorGreen = QgsSettingsEntryInteger( QStringLiteral( "line_color_green" ), QgsSettings::Prefix::QGIS_DIGITIZING, 0 );
-
-    //! Settings entry digitizing line color blue
-    static const inline QgsSettingsEntryInteger settingsDigitizingLineColorBlue = QgsSettingsEntryInteger( QStringLiteral( "line_color_blue" ), QgsSettings::Prefix::QGIS_DIGITIZING, 0 );
-
-    //! Settings entry digitizing line color alpha
-    static const inline QgsSettingsEntryInteger settingsDigitizingLineColorAlpha = QgsSettingsEntryInteger( QStringLiteral( "line_color_alpha" ), QgsSettings::Prefix::QGIS_DIGITIZING, 200 );
+    //! Settings entry digitizing line color
+    static inline QgsSettingsEntryColor *settingsDigitizingLineColor = new QgsSettingsEntryColor( QStringLiteral( "line_color" ), QgsSettings::sTreeDigitizing, QColor( 255, 0, 0, 200 ) );
```

Comment:
What is the reason to instantiate the setting entries with `new`?
I think this one will never be deleted

Reply:
But settings entries are not QObjects, so how is it done?

Reply:
when deleting a tree element, it deletes all it children (tree elements + settings)

![image](https://user-images.githubusercontent.com/127259/210524057-c0108ea1-a389-4bff-878a-44fe13643293.png)

Reply:
Oh thanks i did not see it